terraform/environments/ops/variables.tf

```diff
@@ -27,3 +27,8 @@ variable "repo_name" {
   default     = ""
 }
 
+variable "content_api_url" {
+  type        = string
+  description = "The URL of the `staging` project's content API."
+  default     = ""
```

Comment:
We _could_ try replacing this with a `data` element _in this PR_.

@rogerthatdev thoughts?

Reply:
Let's merge this one like this, and address that next part in the next PR